apps/web/src/app/(signing)/sign/[token]/complete/page.tsx

```diff
@@ -105,15 +109,25 @@ export default async function CompletedSigningPage({
           />
         </div>
 
-        <p className="text-muted-foreground/60 mt-36 text-sm">
-          Want to send slick signing links like this one?{' '}
+        {isLoggedIn ? (
           <Link
-            href="https://documenso.com"
-            className="text-documenso-700 hover:text-documenso-600"
+            href="/documents"
+            className="text-documenso-700 hover:text-documenso-600 mt-36 flex items-center"
           >
-            Check out Documenso.
+            <ChevronLeft className="mr-2 inline-block h-5 w-5" />
+            Return to Homepage
```

Comment:
I'd probably drop the icon and just say "Go Back Home" 😄

Reply:
Done ✅